test/.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+# shellcheck disable=SC1091
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
To explain: running the shell lint checker on this threw a SC1091, file not found, error on `/_/husky.sh`. The alternative to this would be installing husky on the pipeline, but that seemed excessive and unnecessary.